### ISSUE TITLE
Removing certain filters results in a crash

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/nextclade#235]
Removing certain filters causes a hard crash of nextclade. It only occurs on certain filters, and I can't recognize an immediate pattern why some crash things and some work as expected. 

### Steps to reproduce:

1. load the example data & view the tree
2. Click "filter tree"
3. Filter by admin division "Amapa (1)"
4. Click the "x" on the filter badge to remove the filter
5. Hard app crash, with message "An unexpected error has occured". 

### Details

* Firefox 81.0.2.
* Nextclade version 0.7.5 (commit: a0f00c4, build: 738, branch: release)
* Console error:

```
TypeError: a is undefined
    d FilterBadges.tsx:84
    d FilterBadges.tsx:83
    React 2
    Rc FilterBadges.tsx:82
    React 7
    unstable_runWithPriority scheduler.production.min.js:19
    React 15
    unstable_runWithPriority scheduler.production.min.js:19
    React 5
    promise callback*Vu/< _app.tsx:49
    React 2
    unstable_runWithPriority scheduler.production.min.js:19
    React 3
    U scheduler.production.min.js:17
    onmessage scheduler.production.min.js:14
```

**Comment by ivan-aksamentov** [comment]
@jameshadfield Thanks James! Will investigate ASAP.

**Comment by ivan-aksamentov** [comment]
Should be fixed in #236 

It seems that GitHub isn't doing well right now, both, actions and Travis CI don't work. I will check again tomorrow morning and release in 0.7.6 if everything's okay.

**Comment by ivan-aksamentov** [comment]
Released the fix in 0.7.6
